src/errors/error.ts

```diff
@@ -14,10 +14,13 @@ export default class JsonApiError extends Error implements IJsonApiError {
     about?: string;
   };
 
-  constructor(status: HttpStatusCode, code: string) {
+  constructor(status: HttpStatusCode, code: string, detail?: string) {
     super(`${status}: ${code}`);
 
     this.status = status;
     this.code = code;
+    if (detail) {

```

Comment:
@spersico Is there any need for the `if`?

Reply:
noup, I thought that I needed it to avoid a "detail":undefined in the response, but it turns out that it's not necessary. I'll remove it